utils/field.c

```diff
@@ -129,11 +130,17 @@ void setup_field(struct list_head *output_fields, struct opts *opts,
 	if (!strcmp(opts->fields, "none"))
 		return;
 
+	field_flags = calloc(field_table_size, sizeof(bool));
```

Comment:
Please use `xcalloc` to check error.  Also I prefer using `sizeof(*field_flags)` than the type name (bool) because it's easier to deal with future (type) changes.